changelog/5515.feature.rst

```diff
@@ -0,0 +1,10 @@
+Allow selective auto-indentation of multiline log messages.
+
+Adds command line option ``--log-auto-indent``, config option
+``log_auto_indent`` and support for per-entry configuration of
+indentation behavior on calls to ``logging.log()``.
+
+Alters the default for auto-indention from ``on`` to ``off``,
+reverting to the behavior prior to v4.6.0. This reversion to earlier
+behavior was done because it is better to activate new features that
+may lead to broken tests explicitly rather than implicitly.
```

Comment:
Sorry to be annoying, but I think the changelog should clearly mention that this restores the (older) behavior.

Reply:
Can you provide the language you need? I felt that "...reverting to the behavior prior to v4.6.0. This reversion to earlier behavior was done because..." did clearly indicate that we are restoring old behavior.

Reply:
> Sorry to be annoying, but I think the changelog should clearly mention that this restores the (older) behavior.

You're not annoying, you're committed to quality. 

I changed the wording of the changelog to be closer to the wording in your request. Please let me know if I can improve this further, or if there are any other requests you may have.